tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -63,18 +63,33 @@ def setUp(self):
         )
 
     def test_simple(self):
+        with self.feature("organizations:global-views"):
```

Comment:
Do the old events views still work with the new feature flags applied? Those accounts won't have access to new discover.

Reply:
Oof ty, I knew there must have been a reason why this endpoint didn't check global views. Checked and removed the global views check on `events-stats`, `events-meta` and tag value keys which are all still being used by the legacy events view.